src/addremove.js

```diff
@@ -0,0 +1,11 @@
+let tasks = [];
+
+const updatedIndex = () => {
+  tasks = JSON.parse(localStorage.getItem('tasks'));
+  for (let i = 0; i < tasks.length; i += 1) {
+    tasks[i].index = i + 1;
+  }
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+export default updatedIndex;
```

Comment:
:+1: This method works well kudos :muscle:

Reply:
Thank you :)